v3.1.0 Update - WIP

JamesTheClarke opened the pull request:
# Changelog

### Default Modules (Coop)

- CHANGED version number from `v3.0.3` to `v3.1.0`.
CURRENTLY LOCKED FOR COMMITS

---

### Default Modules (PvP)

- CHANGED version number from `v3.0.3` to `v3.1.0`.

---

### BluFor Standard Faction

- ADDED DMT Leader vest item `ACE_AtragMX` - fixes #224 
- ADDED DMT Marksman vest item `ACE_Tripod` - fixes #224 
- CHANGED version number from `v3.0.3` to `v3.1.0`.
- CHANGED GM slotting group name from `GM` to `COMMAND`. - fixes #192
- CHANGED GM map marker name from `GM` to `COMMAND`. - fixes #192
- CHANGED PLT Leader loadout to have identical amount of UGLs as SQD Leaders. - fixes #216
- CHANGED AR ammo type from `rhsusf_100Rnd_556x45_soft_pouch` to `rhsusf_100Rnd_556x45_mixed_soft_pouch`. - fixes #219
- CHANGED AAR backpack ammo type from `rhsusf_100Rnd_556x45_soft_pouch` to `rhsusf_100Rnd_556x45_mixed_soft_pouch`. - fixes #219
- CHANGED Fireteam Supply Box ammo type from `rhsusf_100Rnd_556x45_soft_pouch` to `rhsusf_100Rnd_556x45_mixed_soft_pouch`. - fixes #219
- CHANGED Platoon Supply Crate ammo type from `rhsusf_100Rnd_556x45_soft_pouch` to `rhsusf_100Rnd_556x45_mixed_soft_pouch`. - fixes #219
- CHANGED Fireteam Supply Box ammo type from `rhs_mag_30Rnd_556x45_M855A1_Stanag` to `ACE_30Rnd_556x45_Stanag_Mk318_mag` and `rhs_mag_30Rnd_556x45_M855A1_Stanag_Tracer_Red` to `30Rnd_556x45_Stanag_Tracer_Red`. - fixes #220
- CHANGED Platoon Supply Crate ammo type from `rhs_mag_30Rnd_556x45_M855A1_Stanag` to `ACE_30Rnd_556x45_Stanag_Mk318_mag` and `rhs_mag_30Rnd_556x45_M855A1_Stanag_Tracer_Red` to `30Rnd_556x45_Stanag_Tracer_Red`. - fixes #220
- CHANGED Platoon Supply Crate ammo count `rhsusf_100Rnd_762x51_m61_ap` from 4 to 9 and `rhsusf_100Rnd_762x51_m62_tracer` from 0 to 2. - fixes #220
- CHANGED DMT Spotter ammo type from `rhs_mag_30Rnd_556x45_M855A1_Stanag_No_Tracer` to `ACE_30Rnd_556x45_Stanag_Mk318_mag`. - fixes #220
- CHANGED DMT Marksman rifle from `hlc_rifle_awmagnum` to `rhs_weap_XM2010`. - fixes #218 
- CHANGED DMT night scope from `rhsusf_acc_anpvs27` to `rhsusf_acc_premier_anpvs27`. - fixes #218  
- CHANGED `rankID` for MAT Leader, MMG Leader, ENG Leader, DMT Leader, HAT Leader, HMG, Leader, Mortar Leader from `Sergeant` to `Lieutenant` - fixes #225 
- CHANGED `rankID` for MAT Gunner and MMG Gunner from `Private` to `Corporal` - fixes #225 
- CHANGED init line of resupply crates and boxes to include new textures. - fixes #227
- REMOVED Platoon Supply Crate ammo for SR25 marksman rifle `rhsusf_20Rnd_762x51_SR25_m993_Mag`. - fixes #220

---

### IndFor Standard Faction

- ADDED DMT Leader vest item `ACE_AtragMX` - fixes #224 
- ADDED DMT Marksman vest item `ACE_Tripod` - fixes #224 
- CHANGED version number from `v3.0.3` to `v3.1.0`.
- CHANGED GM slotting group name from `GM` to `COMMAND`. - fixes #192
- CHANGED GM map marker name from `GM` to `COMMAND`. - fixes #192
- CHANGED PLT Leader loadout to have identical amount of UGLs as SQD Leaders. - fixes #216
- CHANGED AR ammo type from `rhsusf_100Rnd_556x45_soft_pouch` to `rhsusf_100Rnd_556x45_mixed_soft_pouch`. - fixes #219
- CHANGED AAR backpack ammo type from `rhsusf_100Rnd_556x45_soft_pouch` to `rhsusf_100Rnd_556x45_mixed_soft_pouch`. - fixes #219
- CHANGED Fireteam Supply Box ammo type from `rhsusf_100Rnd_556x45_soft_pouch` to `rhsusf_100Rnd_556x45_mixed_soft_pouch`. - fixes #219
- CHANGED Platoon Supply Crate ammo type from `rhsusf_100Rnd_556x45_soft_pouch` to `rhsusf_100Rnd_556x45_mixed_soft_pouch`. - fixes #219
- CHANGED Platoon Supply Crate ammo count `hlc_100Rnd_762x51_M_MG3` from 6 to 9 and corrected MMG Gunner machine gun belt from `hlc_100Rnd_762x51_M_M60E4` to `hlc_100Rnd_762x51_M_MG3` - fixes #220
- CHANGED DMT Spotter ammo type from `rhs_mag_30Rnd_556x45_M855A1_Stanag_No_Tracer` to `ACE_30Rnd_556x45_Stanag_Mk318_mag`. - fixes #220
- CHANGED DMT Marksman rifle from `hlc_rifle_awmagnum` to `rhs_weap_XM2010`. - fixes #218  
- CHANGED DMT night scope from `rhsusf_acc_anpvs27` to `rhsusf_acc_premier_anpvs27`. - fixes #218  
- CHANGED `rankID` for MAT Leader, MMG Leader, ENG Leader, DMT Leader, HAT Leader, HMG, Leader, Mortar Leader from `Sergeant` to `Lieutenant` - fixes #225 
- CHANGED `rankID` for MAT Gunner and MMG Gunner from `Private` to `Corporal` - fixes #225 
- CHANGED init line of resupply crates and boxes to include new textures. - fixes #227

---

### OpFor Standard Faction

- ADDED DMT Leader vest item `ACE_AtragMX` - fixes #224 
- ADDED DMT Marksman vest item `ACE_Tripod` - fixes #224 
- CHANGED version number from `v3.0.3` to `v3.1.0`.
- CHANGED GM slotting group name from `GM` to `COMMAND`. - fixes #192
- CHANGED GM map marker name from `GM` to `COMMAND`. - fixes #192
- CHANGED PLT Leader loadout to have identical amount of UGLs as SQD Leaders. - fixes #216
- CHANGED Platoon Supply Crate ammo count `rhs_100Rnd_762x54mmR_7BZ3` from 4 to 9. - fixes #220
- CHANGED `rankID` for MAT Leader, MMG Leader, ENG Leader, DMT Leader, HAT Leader, HMG, Leader, Mortar Leader from `Sergeant` to `Lieutenant` - fixes #225 
- CHANGED `rankID` for MAT Leader, MMG Leader, ENG Leader, DMT Leader, HAT Leader, HMG, Leader, Mortar Leader from `Sergeant` to `Lieutenant` - fixes #225 
- CHANGED `rankID` for MAT Gunner and MMG Gunner from `Private` to `Corporal` - fixes #225
- CHANGED init line of resupply crates and boxes to include new textures. - fixes #227
- CHANGED resupply boxes from white IndFor boxes to BluFor boxes for increased readability. - fixes #227

---

Since there are no balance changes which were not already discussed and agreed upon in a Monthly Meetings only one review by Mission Making Manager Sagu is required. Additional reviews are very much appreciated but not mandatory for a roll-out.

JamesTheClarke committed:
IndFor Standard Faction

- CHANGED version number from v3.0.3 to v3.1.0.
- CHANGED GM slotting group name from `GM` to `Command`. - fixes #192
- CHANGED GM map marker name from `GM` to `CMD`. - fixes #192
- CHANGED PLT Leader loadout to have identical amount of UGLs as SQD Leaders. - fixes #216
- CHANGED AR ammo from `rhsusf_100Rnd_556x45_soft_pouch` to `rhsusf_100Rnd_556x45_mixed_soft_pouch`. - fixes #219
- CHANGED AAR backpack ammo from `rhsusf_100Rnd_556x45_soft_pouch` to `rhsusf_100Rnd_556x45_mixed_soft_pouch`. - fixes #219
- CHANGED Fireteam Supply Box ammo from `rhsusf_100Rnd_556x45_soft_pouch` to `rhsusf_100Rnd_556x45_mixed_soft_pouch`. - fixes #219
- CHANGED Platoon Supply Crate ammo from `rhsusf_100Rnd_556x45_soft_pouch` to `rhsusf_100Rnd_556x45_mixed_soft_pouch`. - fixes #219

CNTODachi committed:
US DMT Marksman Rifle CHANGED

CHANGED US DMT Marksman rifle from the "hlc_rifle_awmagnum" to the "rhs_weap_XM2010".

JamesTheClarke committed:
BluFor Standard Faction Update

- CHANGED version number from v3.0.3 to v3.1.0.
- CHANGED GM slotting group name from `GM` to `COMMAND`. - fixes #192
- CHANGED GM map marker name from `GM` to `COMMAND`. - fixes #192
- CHANGED PLT Leader loadout to have identical amount of UGLs as SQD Leaders. - fixes #216
- CHANGED AR ammo from `rhsusf_100Rnd_556x45_soft_pouch` to `rhsusf_100Rnd_556x45_mixed_soft_pouch`. - fixes #219
- CHANGED AAR backpack ammo from `rhsusf_100Rnd_556x45_soft_pouch` to `rhsusf_100Rnd_556x45_mixed_soft_pouch`. - fixes #219
- CHANGED Fireteam Supply Box ammo from `rhsusf_100Rnd_556x45_soft_pouch` to `rhsusf_100Rnd_556x45_mixed_soft_pouch`. - fixes #219
- CHANGED Platoon Supply Crate ammo from `rhsusf_100Rnd_556x45_soft_pouch` to `rhsusf_100Rnd_556x45_mixed_soft_pouch`. - fixes #219
- CHANGED Fireteam Supply Box ammo from `rhs_mag_30Rnd_556x45_M855A1_Stanag` to `ACE_30Rnd_556x45_Stanag_Mk318_mag` and `rhs_mag_30Rnd_556x45_M855A1_Stanag_Tracer_Red` to `30Rnd_556x45_Stanag_Tracer_Red`. - fixes #220
- CHANGED Platoon Supply Crate ammo from `rhs_mag_30Rnd_556x45_M855A1_Stanag` to `ACE_30Rnd_556x45_Stanag_Mk318_mag` and `rhs_mag_30Rnd_556x45_M855A1_Stanag_Tracer_Red` to `30Rnd_556x45_Stanag_Tracer_Red`. - fixes #220

Seb105 commented:
Issue #220 applies to TFN too. I did fix this before it was deleted 🤷‍♂️

JamesTheClarke committed:
OpFor Standard Faction Update

- CHANGED version number from v3.0.3 to v3.1.0.
- CHANGED GM slotting group name from `GM` to `COMMAND`. - fixes #192
- CHANGED GM map marker name from `GM` to `COMMAND`. - fixes #192
- CHANGED PLT Leader loadout to have identical amount of UGLs as SQD Leaders. - fixes #216

JamesTheClarke commented:
> Issue #220 applies to TFN too. I did fix this before it was deleted 🤷‍♂️

Is this issue present in TFN outside the boxes? I only checked the Fireteam box and the Platoon crate as per issue description. Both have `ACE_30Rnd_556x45_Stanag_Mk318_mag` and `30Rnd_556x45_Stanag_Tracer_Red`.

Seb105 commented:
I did a simple find and replace in the .sqm rather than checking in game, but it did pick something up in TFN, though I'm not sure which box exactly (it may have been a unit too)

You might be able to figure it out from my commit by lookig at the of lines affected  as all our crates and units are named it will give you a clue in the .sqm

JamesTheClarke commented:
> I did a simple find and replace in the .sqm rather than checking in game, but it did pick something up in TFN, though I'm not sure which box exactly (it may have been a unit too)
> 
> You might be able to figure it out from my commit by lookig at the of lines affected as all our crates and units are named it will give you a clue in the .sqm

Cheers, I'll tag the issue #220 with TFN and make a checkbox note to keep track of this for tomorrow.

JamesTheClarke commented:
Quick status report for all GitHub contributors:

- [x] BluFor, IndFor and OpFor compositions are open for commits.

- [ ] Default Modules Coop & PVP are currently locked for commits as I still need to double-check a commit for each that I have in the works for a while now. Please don't commit to those two compositions as they are likely to be overwritten. ETA for unlock: Wednesday night 20.07.2022.

CNTODachi committed:
US DMT Marksman Rifle CHANGED

CHANGED US DMT Marksman rifle from the "hlc_rifle_awmagnum" to the "rhs_weap_XM2010".

CNTODachi committed:
REMOVED SR-25 Mags from US PLT Crate

REMOVED SR-25 Mags from US PLT Crate as they are not used.

CNTODachi committed:
CHANGED TFN DMT night scope for proper sniper scope.

CHANGED TFN DMT night scope from the "rhsusf_acc_anpvs27" to the "rhsusf_acc_premier_anpvs27".

JamesTheClarke committed:
BluFor MMG Platoon Resupply Crate

- CHANGED Platoon Supply Crate ammo count `rhsusf_100Rnd_762x51_m61_ap` from 4 to 9 and `rhsusf_100Rnd_762x51_m62_tracer` from 0 to 2. - fixes #220

JamesTheClarke committed:
IndFor MMG Platoon Resupply Crate

- CHANGED Platoon Supply Crate ammo count `hlc_100Rnd_762x51_M_MG3` from 6 to 9 and corrected MMG Gunner machine gun belt from 'hlc_100Rnd_762x51_M_M60E4' to 'hlc_100Rnd_762x51_M_MG3' - fixes #220

JamesTheClarke committed:
OpFor MMG Platoon Resupply Crate

- CHANGED Platoon Supply Crate ammo count `rhs_100Rnd_762x54mmR_7BZ3` from 4 to 9. - fixes #220

JamesTheClarke committed:
IndFor DMT Spotter STANAG Mags

- CHANGED DMT Spotter ammo type from `rhs_mag_30Rnd_556x45_M855A1_Stanag_No_Tracer` to `ACE_30Rnd_556x45_Stanag_Mk318_mag`. - fixes #220

JamesTheClarke committed:
BluFor DMT Spotter STANAG Mags

- CHANGED DMT Spotter ammo type from `rhs_mag_30Rnd_556x45_M855A1_Stanag_No_Tracer` to `ACE_30Rnd_556x45_Stanag_Mk318_mag`. - fixes #220

JamesTheClarke committed:
RankID Fixes for BluFor, IndFor & OpFor

- CHANGED rankID for MAT Leader, MMG Leader, ENG Leader, DMT Leader, HAT Leader, HMG, Leader, Mortar Leader from "Sergeant" to "Lieutenant". fixes #225
- CHANGED rankID MAT Gunner & MMG Gunner from "Private" to "Corporal" to ensure correct element leadership transfer when leader dies (equivalent to Dagger).

JamesTheClarke committed:
DMT Additional Equipment for BluFor, IndFor & OpFor

- ADDED DMT Leader vest item "ACE_AtragMX" - fixes #224
- ADDED DMT Marksman vest item "ACE_Tripod" - fixes #224

JamesTheClarke committed:
New Resupply Crate Textures for BluFor, IndFor & OpFor

- CHANGED init line of crates and boxes for BluFor. - fixes #227
- CHANGED init line of crates and boxes for IndFor. - fixes #227
- CHANGED init line of crates and boxes for OpFor. - fixes #227
- CHANGED OpFor composition boxes from white InFor boxes to BluFor boxes for increased readability. - fixes #227

LongbeardNor commented:
What is missing before this update can be pushed?

LongbeardNor commented:
Can we have the current fixed merged, and the stuff that is not done switched to a new pull request?
